utils/_context/containers.py

```diff
@@ -555,12 +555,23 @@ def __init__(
 
         self.port = weblog.port
 
+        volumes = {
+            f"./{host_log_folder}/docker/weblog/logs/": {"bind": "/var/log/system-tests", "mode": "rw",},
+        }
+
+        if os.path.exists("./binaries/nodejs-load-from-local"):
+            path = open("./binaries/nodejs-load-from-local").read().strip(" \r\n")
```

Comment:
use a context manager to ensure that the file is closed in any situation (and `strip`, by default, strips any spaces/newline chars) : 

```suggestion
            with open("./binaries/nodejs-load-from-local", encoding="utf-8") as f:
                path = f.read().strip()
```

Reply:
Are you sure about newlines? I see other occurrences of this type of stripping in the codebase, and the documentation is also unclear on that.

Reply:
I'm pretty sure, and I even tested before commenting :) 

It removes `whitespaces`, see this explanation : https://stackoverflow.com/questions/77258198/full-list-symbols-stripped-with-str-strip-by-default

The [doc](https://docs.python.org/3.4/library/stdtypes.html#str.strip) is a little bit misleading, as it does not precise that "whitespace" is this list of chars.

![image](https://github.com/user-attachments/assets/fc026813-c7f7-4d93-bceb-a11e3298660e)

But nevermind, that'a a detail.

Reply:
Though, I'm curious about the test you perform, is there a situation where it does not work ? 🤔